cmake/modules/FindGLM.cmake

```diff
@@ -56,10 +56,6 @@ INCLUDE(FindPackageHandleStandardArgs)
 FIND_PACKAGE_HANDLE_STANDARD_ARGS(GLM DEFAULT_MSG
     GLM_INCLUDE_DIR)
 
-IF(GLM_FOUND)
-    SET(GLM_INCLUDE_DIRS "${GLM_INCLUDE_DIR}")
+SET(GLM_INCLUDE_DIRS "${GLM_INCLUDE_DIR}")
 
-    IF(NOT GLM_FIND_QUIETLY)
-        MESSAGE(STATUS "GLM_INCLUDE_DIR = ${GLM_INCLUDE_DIR}")
-    ENDIF(NOT GLM_FIND_QUIETLY)
-ENDIF(GLM_FOUND)
+mark_as_advanced(GLM_INCLUDE_DIR)
```

Comment:
I suggest using a file you find elsewhere for this. GLM is a pretty common lib. CMake depends on standardization. Often cmake files are promoted to being included in newer CMake versions.

Also note that GLM used to come with it's own cmake file apparently (which we should probably use as it's the most likely to be included in CMake upstream): https://chromium.googlesource.com/external/github.com/g-truc/glm/+/0.9.0/FindGLM.cmake (not sure if there are more recent versions or if/why it was removed)